File: Lakshay Garg/Leetcode/Two_Pointers_Leetcode/Remove_Element.cpp
class Solution {
public:
    int removeElement(vector<int>& v, int val) {
        int n=v.size();
        int s=0;
        for(int i=0;i<n;i++){
           if(v[i]!=val){
                v[s]=v[i];
                s++;
            }
        }
        return s;
    }
};

File: Anya_Seth/2-Pointer/Remove Element.java
class Solution {
    public int removeElement(int[] arr, int val) {
        int j = 0;
        for(int i = 0 ; i < arr.length ; i++){
            if(arr[i] != val){
                int temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
                j++;
            }
        }
        int count = 0;
        for(int i = 0 ; i < arr.length ; i++){
            if(arr[i] != val){
                count++;
            }
        }
        return count;
    }
}
